x-pack/filebeat/input/azureblobstorage/input.go

```diff
@@ -0,0 +1,150 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build !aix
+// +build !aix
+
+package azureblobstorage
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	v2 "github.com/elastic/beats/v7/filebeat/input/v2"
+	cursor "github.com/elastic/beats/v7/filebeat/input/v2/input-cursor"
+	"github.com/elastic/beats/v7/libbeat/feature"
+	"github.com/elastic/beats/v7/x-pack/filebeat/input/azureblobstorage/state"
+	"github.com/elastic/beats/v7/x-pack/filebeat/input/azureblobstorage/types"
+	conf "github.com/elastic/elastic-agent-libs/config"
+	"github.com/elastic/elastic-agent-libs/logp"
+)
+
+type azurebsInput struct {
+	config     config
+	serviceURL string
+}
+
+const (
+	inputName string = "azureblobstorage"
```

Comment:
I think `azure-blob-storage` is more in line with naming of other inputs. For example we have `aws-s3` and `gcp-pubsub`.

Reply:
sure , will update this